node/consensus/src/lib.rs

```diff
@@ -247,9 +249,18 @@ impl<N: Network> Consensus<N> {
             if self.ledger.contains_transmission(&TransmissionID::from(&transaction_id))? {
                 bail!("Transaction '{}' exists in the ledger {}", fmt_id(transaction_id), "(skipping)".dimmed());
             }
-            // Add the transaction to the memory pool.
+            // Get the priority_fee amount from the Deploy or Execute Transaction. Fee Transactions are not expected here.
+            let priority_fee = match &transaction {
```

Comment:
since this is only used for queuing, I wonder if it wouldn't be cleaner if we obtained the fee within `add_transaction_to_queue`, which would only need the `Transaction` as a parameter